starknet_py/tests/e2e/docs/code_examples/test_contract.py

```diff
@@ -23,7 +23,7 @@ def test_init():
         ],
         provider=Account(
             address=0x321,
-            client=GatewayClient(TESTNET),
```

Comment:
I'm not sure about defaulting to full node client in docs already.

cc @war-in @drknzz

Reply:
Right, it should be a separate PR, after 0.16, to default to FullNodeClient in the docs.

Reply:
Created an issue targeting the problem.